src/implementations/twig/components/file/file.html.twig

```diff
@@ -160,33 +162,49 @@
     {% endif %}
     </div>
   </div>
+
   <div class="ecl-file__footer">
   {% if _language is not empty %}
-    <div class="ecl-file__language" id={{ _lang_id }}>{{ _language }}</div>
+    <div class="ecl-file__language" id="{{ _lang_id }}">{{ _language }}</div>
   {% endif %}
   {% if _meta is not empty %}
     <div class="ecl-file__meta">{{ _meta }}</div>
   {% endif %}
-    {% include '@ecl/link/link.html.twig' with _download|merge({
-      link: _download.link|default({})|merge({
-        type: 'standalone',
-        icon_position: 'after',
-      }),
-      icon: _download.icon|default({})|merge({
-        name: 'download',
-        size: 'fluid'
-      }),
-      extra_attributes: [
-       { name: 'id', value: _link_id },
-       { name: 'aria-labelledby', value: _label_id ~ ' ' ~ _title_id ~ ' ' ~ _lang_id },
-      ],
-      extra_classes: 'ecl-file__download',
-    }) only %}
+    <div class="ecl-file__action">
+      {% set _download_link_attributes = _download.extra_attributes|default([])|merge([
+        { name: 'id', value: _download_id },
+        { name: 'aria-labelledby', value: _download_label_id ~ ' ' ~ _title_id ~ ' ' ~ _lang_id },
+      ]) %}
+      {% if _download_attribute %}
+        {% set _download_link_attributes = _download_link_attributes|merge([
+          { name: 'download' },
+        ]) %}
+      {% endif %}
+
+      {% set _download_link_classes = 'ecl-file__download' %}
+      {% if _download.extra_classes is not empty %}
+        {% set _download_link_classes = _download_link_classes ~ ' ' ~ _download.extra_classes %}
+      {% endif %}
+
+      {% include '@ecl/link/link.html.twig' with _download|merge({
+        link: _download.link|default({})|merge({
+          type: 'standalone',
+          icon_position: 'after',
+        }),
+        icon: _download.icon|default({})|merge({
+          name: 'download',
+          size: 'fluid'
+        }),
+        extra_attributes: _download_link_attributes,
+        extra_classes: _download_link_classes,
+      }) only %}
+    </div>
   </div>
+
 {% if _translation is not empty %}
   {% include '@ecl/file/file-translations.html.twig' with _translation|merge({
     id: _id,
-    variant: _variant,
+    download_attribute: _download_attribute,
```

Comment:
basically here we are deciding that if there is a download attribute on the "main" file this will also be applied to all the translations and there is no chance of limiting this to the main one.
Might be this the expected behaviour and surely setting it once it's easier than doing it for each item, but the way i feel is that if we enable a "feature" we should also give a way to "disable" it.

What i would do is still to pass this value but not as the download_attribute and then use this as the default value for the download_attribute in the item template, this way you could still set download_attribute to false (or true) on a single translation overriding the value coming from the main "link".